src/users/aliases.md

```diff
@@ -0,0 +1,27 @@
+# Aliases
+
+Aliases are individual emojis that you can name and then be used through that name.
+Aliases are user dependent, meaning that only the user that named or imported one can use it through that name. 
+For more information regarding how to name server wide aliases refer to [adding emojis through packs and aliases](../guild/Through_packs_and_aliases.md).
+
+You can find the full list of Aliases you have through the use of the command `/alias mine` or `/alias dm`(same response but sent to dm)
```

Comment:
`alias mine` shows a paginated list, but `alias dm` sends the full list in multiple messages